src/Shared/Recruit.Vacancies.Client/Infrastructure/EventHandlers/UpdateEmployerDashboardOnChange.cs

```diff
@@ -95,6 +96,17 @@ public Task Handle(VacancyReferredEvent notification, CancellationToken cancella
             return Handle(notification);
         }
 
+        public async Task Handle(VacancyTransferredEvent notification, CancellationToken cancellationToken)
```

Comment:
This could result in rebuilding dashboard many times with in a short span, but I am not sure what the alternative is.

Reply:
Yeah not ideal. I have mentioned this to Mark and he knew this was likely to happen. We will just have to see how it copes. Assuming this process isn't happening too often and on large numbers of vacancies makes it easier to live with for now.